frontend/test/components/RosalutionHeader.spec.js

```diff
@@ -98,4 +98,68 @@ describe('RosalutionHeaderComponent.vue', () => {
 
     expect(wrapper.vm.$route.path).to.equal('/rosalution/account');
   });
+
+  it('should display the proper workflow icon based on the workflow_status prop', async () => {
+    const wrapper = getMountedComponent({workflow_status: 'Annotation'});
+    let statusIcons = wrapper.findAllComponents({name: 'FontAwesomeIcon'});
+    let statusIcon = statusIcons[0];
+    expect(statusIcon.props('icon')).to.equal('asterisk');
```

Comment:
To streamline testing of each case,  it is better to try and parameterize the tests so that the same code isn't copied many times.  An example of this is done in the **AnalysisCard.spec.js** tests here https://github.com/uab-cgds-worthey/rosalution/blob/abea2a5f0d76ccaa16bd443ead4a895c71db12ec/frontend/test/components/AnalysisListing/AnalysisCard.spec.js#L86.  

You don't need to test each case also, its best to try and choose the few cases to verify that the core feature is working, so that if something slightly changes, it doesn't break the tests.